pages/data-design/avo-tracking-plan/events.mdx

```diff
@@ -20,10 +27,20 @@ An event is defined in the Events section of the Tracking plan and can be organi
 - [_Sources_](/data-design/define-sources-and-destinations#sources-where-the-data-comes-from) that the event should be sent from
 - [_Actions_](#actions) associated with the event, including properties
 - [_Metrics_](/data-design/avo-tracking-plan/metrics) related to the event (optional)
+- [Stakeholders](/data-design/avo-tracking-plan/stakeholder-domains) in your organization that rely on the event
 - [_Categories_](/data-design/organizing-metrics-and-events#categories) that the event is a part of (optional)
 - _Tags_ associated with the event
 - [_Name Mapping_](/data-design/name-mapping) that the event name is mapped to when sending to specific destinations
 
+You can define events in several ways, depending on the scope of your changes and level of detail you need:
+
+1. **Inline Table Editing**: Properties, categories and tags can be edited directly in the table by clicking on individual cells.
+
+3. **Batch Editing**: Select multiple events to apply bulk updates to shared attributes like sources, properties, and stakeholders.
+
+1. **Event Details Panel**: Click into an event to access and define all aspects of the event.
```

Comment:
Should these be bullets rather than counters? And should we move the event details to the top as it's the default way to edit events?

Reply:
My understanding (and what makes sense to me) is that we're shifting the default way to be editing from the table (e.g. by not opening the details by default). So this order makes sense to me